src/cloud/docker/docker-containers-service.md

```diff
@@ -0,0 +1,202 @@
+
+## Database container
+
+**Container name:** db<br/>
+**Docker base image:** [mariadb]<br/>
+**Ports exposed:** 3306<br/>
+
+The Database container is based on the [mariadb] image and includes the following volumes:
+
+-  `magento-db: /var/lib/mysql`
+-  `.docker/mysql/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d`
+
+When a database container initializes, it creates a new database with the specified name and uses the configuration variables specified in the docker-compose configuration. The initial start-up process also executes files with `.sh`, `.sql` and `.sql.gz` extensions that are found in the `/docker-entrypoint-initdb.d` directory. Files are executed in alphabetical order. See [mariadb Docker documentation](https://hub.docker.com/_/mariadb).
+
+To prevent accidental data loss, the database is stored in a persistent **`magento-db`** volume after you stop and remove the Docker configuration. The next time you use the `docker-compose up` command, the Docker environment restores your database from the persistent volume. You must manually destroy the database volume using the `docker volume rm <volume_name>` command.
+
+You can inject a MySQL configuration into the database container at creation by adding the configuration to the `docker-compose-override.yml` file. Add the custom values using an included `my.cnf` file, or add the correct variables directly to the override file as shown in the following examples.
+
+Add a custom `my.cnf` file to the `docker-compose.override.yml` file:
+
+```yaml
+db:
+  volumes:
+    - path/to/custom.my.cnf:/etc/mysql/conf.d/custom.my.cnf
+```
+
+Add configuration values to the `docker-compose.override.yml` file:
+
+```yaml
+  db:
+    environment:
+      - innodb-buffer-pool-size=134217728
+```
+
+See [Manage the database] for details about using the database.
+
+## Elasticsearch container
+
+**Container name:** elasticsearch<br/>
+**Docker base image:** [magento/magento-cloud-docker-elasticsearch](https://hub.docker.com/r/magento/magento-cloud-docker-elasticsearch)<br/>
+**Ports exposed:** 9200, 9300<br/>
+
+The Elasticsearch container for {{site.data.var.mcd}} is a standard Elasticsearch container with required plugins and configurations for {{site.data.var.ee}}.
+
+## FPM container
+
+**Container name:** fpm<br/>
+**Docker base image:** [magento/magento-cloud-docker-php][php-cloud], which is based on the [php](https://hub.docker.com/_/php) Docker image<br/>
+**Ports exposed:** 9000, 9001<br/>
+
+The FPM container includes the following volumes:
+
+-  Read-only volumes:
+   -  `/app`
+   -  `/app/vendor`
+   -  `/app/generated`
+   -  `/app/setup`
+
+-  Read/Write volumes:
+   -  `/app/var`
+   -  `/app/app/etc`
+   -  `/app/pub/static`
+   -  `/app/pub/media`
+
+You can load custom extensions in the FPM configuration by adding the configuration to the `docker-compose.override.yml` file. This configuration is applied when you build and deploy.
```

Comment:
You can also manage those through .magento.app.yaml. Need to clarify that too because if you can natively load extension through config file, editing overrides would not be required.